packages/auth/test/base-test.ts

```diff
@@ -21,9 +21,15 @@ export abstract class BaseTest {
           name: 'Bilu',
           role: UserRole.admin,
         }),
-        getCompany: () => ({
-          id: '114',
-        }),
+        getCompany: (id?: string) => {
+          if (id && id !== '114') {
+            return null
+          }
```

Comment:
I think you don't need to check if the value is not null
```suggestion
          if (id !== '114') return null
```